experimental/generation/generator/templates/professional/Relationship.qna

```diff
@@ -0,0 +1,99 @@
+> Collection of chit chat interactions between user and bot. You can add additional questions or adapt the answer as you see fit. Here's the source editorial dataset - [Source](https://github.com/Microsoft/BotBuilder-PersonalityChat/blob/master/CSharp/Datasets/Queries_Responses_Professional_QnAMaker.tsv)
+
```

Comment:
All of the .qna files should be X.en-us.qna--then you don't need the special case any more. They should also be in an en-us subdirectory.  There are 5 styles and 9 languages for .qna we will bring in here soon as well as If you haven't used it the new powertoys has a nice renaming tool.